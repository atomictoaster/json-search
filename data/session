a
3
.h
id
n
_id
.b
tags
.h
Mulino
.b
role
end-user
.more
.done
.back
signature
Unicode ist spa√ü
