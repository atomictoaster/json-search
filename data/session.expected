JSON Search Tool

Please select a dataset to search, or '.quit' to exit:
   1) Organizations 2) Tickets 3) Users 
# 
Invalid selection: 'a'.  Enter the number of a dataset instead.

Please select a dataset to search, or '.quit' to exit:
   1) Organizations 2) Tickets 3) Users 
# 


   '.help|.h' to see available fields
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a term to search Users # 
User records contain the following fields:
* _id
* active
* alias
* created_at
* email
* external_id
* last_login_at
* locale
* name
* organization_id
* phone
* role
* shared
* signature
* suspended
* tags
* timezone
* url
* verified

   '.help|.h' to see available fields
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a term to search Users # Key 'id' is not an expected key for Users. Continue anyway (Yn)? 

   '.help|.h' to see available fields
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a term to search Users # 
   '.help|.h' to see example values
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a value to search Users[_id] # 
   '.help|.h' to see available fields
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a term to search Users # 
   '.help|.h' to see example values
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a value to search Users[tags] # 
User records contain 'tags' values like:
* Springville
* Sutton
* Hartsville/Hartley
* Diaperville
* Foxworth
* Woodlands
* Herlong
* Henrietta
* Mulino
* Kenwood
* Wescosville
* Loyalhanna
* Gallina
* Glenshaw
* Rowe
* Babb

   '.help|.h' to see example values
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a value to search Users[tags] # Searching 75 records for entries with 'tags' equal to 'Mulino'
1 record(s) 

**** Result[1/1]
                    _id:  3
                 active:  false
                  alias:  Miss Buck
             created_at:  2016-07-28T05:29:25 -10:00
                  email:  buckwagner@flotonic.com
            external_id:  85c599c1-ebab-474d-a4e6-32f1c06e8730
          last_login_at:  2013-02-07T05:53:38 -11:00
                 locale:  en-AU
                   name:  Ingrid Wagner
        organization_id:  104
                  phone:  9365-482-943
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Mulino
                tags[1]:  Kenwood
                tags[2]:  Wescosville
                tags[3]:  Loyalhanna
               timezone:  Trinidad and Tobago
                    url:  http://initech.zendesk.com/api/v2/users/3.json
               verified:  false

End of results

   '.help|.h' to see example values
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a value to search Users[tags] # 
   '.help|.h' to see available fields
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a term to search Users # 
   '.help|.h' to see example values
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a value to search Users[role] # Searching 75 records for entries with 'role' equal to 'end-user'
26 record(s) 

**** Result[1/26]
                    _id:  3
                 active:  false
                  alias:  Miss Buck
             created_at:  2016-07-28T05:29:25 -10:00
                  email:  buckwagner@flotonic.com
            external_id:  85c599c1-ebab-474d-a4e6-32f1c06e8730
          last_login_at:  2013-02-07T05:53:38 -11:00
                 locale:  en-AU
                   name:  Ingrid Wagner
        organization_id:  104
                  phone:  9365-482-943
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Mulino
                tags[1]:  Kenwood
                tags[2]:  Wescosville
                tags[3]:  Loyalhanna
               timezone:  Trinidad and Tobago
                    url:  http://initech.zendesk.com/api/v2/users/3.json
               verified:  false

**** Result[2/26]
                    _id:  4
                 active:  true
                  alias:  Mr Cardenas
             created_at:  2016-02-09T07:52:10 -11:00
                  email:  cardenasnewton@flotonic.com
            external_id:  37c9aef5-cf01-4b07-af24-c6c49ac1d1c7
          last_login_at:  2012-09-25T01:32:46 -10:00
                 locale:  de-CH
                   name:  Rose Newton
        organization_id:  122
                  phone:  8685-482-450
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  true
                tags[0]:  Gallina
                tags[1]:  Glenshaw
                tags[2]:  Rowe
                tags[3]:  Babb
               timezone:  Netherlands
                    url:  http://initech.zendesk.com/api/v2/users/4.json
               verified:  true

**** Result[3/26]
                    _id:  6
                 active:  false
                  alias:  Miss Shelton
             created_at:  2016-04-04T01:30:49 -10:00
                  email:  sheltonhebert@flotonic.com
            external_id:  ed106e63-396d-4d16-ae49-d3dd37049ba3
          last_login_at:  2012-08-24T10:20:58 -10:00
                 locale:  zh-CN
                   name:  Riggs Hebert
        organization_id:  109
                  phone:  8935-413-112
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Belfair
                tags[1]:  Chamberino
                tags[2]:  Roberts
                tags[3]:  Cascades
               timezone:  Liberia
                    url:  http://initech.zendesk.com/api/v2/users/6.json
               verified:  false

**** Result[4/26]
                    _id:  10
                 active:  true
                  alias:  Mr Webb
             created_at:  2016-02-08T04:32:38 -11:00
                  email:  webbvinson@flotonic.com
            external_id:  f744706e-df3d-4d51-ad18-c63e41be5cc0
          last_login_at:  2014-12-26T04:54:30 -11:00
                 locale:  en-AU
                   name:  Kari Vinson
        organization_id:  105
                  phone:  8235-883-140
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Moraida
                tags[1]:  Beechmont
                tags[2]:  Jeff
                tags[3]:  Starks
               timezone:  Germany
                    url:  http://initech.zendesk.com/api/v2/users/10.json
               verified:  false

**** Result[5/26]
                    _id:  12
                 active:  false
                  alias:  Mr Sally
             created_at:  2016-07-21T12:26:16 -10:00
                  email:  sallyhammond@flotonic.com
            external_id:  38899b1e-89ca-43e7-b039-e3c88525f0d2
          last_login_at:  2012-12-29T07:59:56 -11:00
                 locale:  en-AU
                   name:  Watkins Hammond
        organization_id:  110
                  phone:  8144-293-283
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Bonanza
                tags[1]:  Balm
                tags[2]:  Fulford
                tags[3]:  Austinburg
               timezone:  United Kingdom
                    url:  http://initech.zendesk.com/api/v2/users/12.json
               verified:  false

**** Result[6/26]
                    _id:  13
                 active:  true
                  alias:  Mr Vera
             created_at:  2016-06-02T01:11:48 -10:00
                  email:  veramoore@flotonic.com
            external_id:  dce36d80-f143-4b1e-8a2e-47651e367a4e
          last_login_at:  2012-08-03T05:09:18 -10:00
                 locale:  en-AU
                   name:  Wade Moore
        organization_id:  112
                  phone:  8395-652-590
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  true
                tags[0]:  Bartonsville
                tags[1]:  Frystown
                tags[2]:  Sparkill
                tags[3]:  Dola
               timezone:  Saudi Arabia
                    url:  http://initech.zendesk.com/api/v2/users/13.json
               verified:  true

**** Result[7/26]
                    _id:  20
                 active:  false
                  alias:  Miss Hayes
             created_at:  2016-04-07T11:08:24 -10:00
                  email:  hayesdavidson@flotonic.com
            external_id:  1c8b9696-b5c5-4f5a-b077-5d85dd8b22f4
          last_login_at:  2014-05-04T09:18:56 -10:00
                 locale:  en-AU
                   name:  Lee Davidson
        organization_id:  104
                  phone:  9544-832-980
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  true
                tags[0]:  Riegelwood
                tags[1]:  Snyderville
                tags[2]:  Roland
                tags[3]:  National
               timezone:  Norway
                    url:  http://initech.zendesk.com/api/v2/users/20.json
               verified:  false

**** Result[8/26]
                    _id:  22
                 active:  true
                  alias:  Miss Cohen
             created_at:  2016-01-28T08:49:17 -11:00
                  email:  cohenvincent@flotonic.com
            external_id:  bf313bac-e4b1-46a3-a7ee-7cc584b7cbb8
          last_login_at:  2014-07-30T03:32:49 -10:00
                 locale:  en-AU
                   name:  Russo Vincent
        organization_id:  124
                  phone:  8895-892-293
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Veyo
                tags[1]:  Highland
                tags[2]:  Kiskimere
                tags[3]:  Hoehne
               timezone:  Qatar
                    url:  http://initech.zendesk.com/api/v2/users/22.json
               verified:  false

**** Result[9/26]
                    _id:  26
                 active:  false
                  alias:  Miss Jones
             created_at:  2016-04-23T06:34:15 -10:00
                  email:  joneswright@flotonic.com
            external_id:  2fe9f6de-4576-4076-9686-19d7cfcc8378
          last_login_at:  2014-12-24T02:32:32 -11:00
                 locale:  zh-CN
                   name:  Rosa Wright
        organization_id:  123
                  phone:  8034-232-199
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  true
                tags[0]:  Kidder
                tags[1]:  Castleton
                tags[2]:  Stewartville
                tags[3]:  Bascom
                    url:  http://initech.zendesk.com/api/v2/users/26.json
               verified:  false

**** Result[10/26]
                    _id:  29
                 active:  false
                  alias:  Mr Vance
             created_at:  2016-03-17T06:09:57 -11:00
                  email:  vancenorman@flotonic.com
            external_id:  5cf7c032-b3cb-4c87-afa1-57fc9f94e9a1
          last_login_at:  2016-05-17T03:03:05 -10:00
                 locale:  en-AU
                   name:  Herrera Norman
        organization_id:  101
                  phone:  9444-743-342
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Tilden
                tags[1]:  Layhill
                tags[2]:  Franklin
                tags[3]:  Allensworth
               timezone:  Zimbabwe
                    url:  http://initech.zendesk.com/api/v2/users/29.json
               verified:  false

   .done|.d to stop
   .quit|.q to exit
   or enter to continue

17 results remaining # 
**** Result[11/26]
                    _id:  31
                 active:  true
                  alias:  Miss Carroll
             created_at:  2016-01-10T05:55:31 -11:00
                  email:  carrollraymond@flotonic.com
            external_id:  e9121c21-0321-4024-a844-5d869ddc1343
          last_login_at:  2016-06-06T04:20:09 -10:00
                 locale:  zh-CN
                   name:  Jessica Raymond
        organization_id:  104
                  phone:  9794-023-473
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  true
                tags[0]:  Chalfant
                tags[1]:  Calverton
                tags[2]:  Flintville
                tags[3]:  Yonah
               timezone:  Greece
                    url:  http://initech.zendesk.com/api/v2/users/31.json
               verified:  false

**** Result[12/26]
                    _id:  36
                 active:  false
                  alias:  Mr Nielsen
             created_at:  2016-07-28T04:55:50 -10:00
                  email:  nielsennash@flotonic.com
            external_id:  4a0e0eda-f027-4727-910f-148d2b7ab103
          last_login_at:  2013-06-08T01:53:28 -10:00
                 locale:  zh-CN
                   name:  Craig Nash
        organization_id:  115
                  phone:  8894-673-477
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Independence
                tags[1]:  Bainbridge
                tags[2]:  Goodville
                tags[3]:  Gasquet
               timezone:  Swaziland
                    url:  http://initech.zendesk.com/api/v2/users/36.json
               verified:  true

**** Result[13/26]
                    _id:  40
                 active:  true
                  alias:  Mr Neal
             created_at:  2016-07-16T09:13:47 -10:00
                  email:  nealengland@flotonic.com
            external_id:  e79e166e-fcb5-4604-9466-1ea9777c6eb5
          last_login_at:  2015-10-20T01:25:19 -11:00
                 locale:  zh-CN
                   name:  Burgess England
        organization_id:  113
                  phone:  9675-282-161
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  true
                tags[0]:  Riceville
                tags[1]:  Ribera
                tags[2]:  Caberfae
                tags[3]:  Breinigsville
               timezone:  Taiwan
                    url:  http://initech.zendesk.com/api/v2/users/40.json
               verified:  false

**** Result[14/26]
                    _id:  41
                 active:  false
                  alias:  Miss Janet
             created_at:  2016-01-12T10:26:24 -11:00
                  email:  janetblack@flotonic.com
            external_id:  497def74-ce88-442e-814c-2fb8ce945dd9
          last_login_at:  2015-02-21T08:07:14 -11:00
                 locale:  en-AU
                   name:  Alvarez Black
        organization_id:  106
                  phone:  9464-452-234
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Bend
                tags[1]:  Troy
                tags[2]:  Fedora
                tags[3]:  Celeryville
               timezone:  Finland
                    url:  http://initech.zendesk.com/api/v2/users/41.json
               verified:  false

**** Result[15/26]
                    _id:  43
                 active:  true
                  alias:  Mr Kaitlin
             created_at:  2016-02-03T07:44:35 -11:00
                  email:  kaitlincoleman@flotonic.com
            external_id:  08a71e65-dd92-4774-966a-876766c0f962
          last_login_at:  2012-10-29T10:06:17 -11:00
                 locale:  en-AU
                   name:  Charlene Coleman
        organization_id:  122
                  phone:  8204-383-661
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Wyano
                tags[1]:  Soudan
                tags[2]:  Coinjock
                tags[3]:  Mathews
               timezone:  Maldives
                    url:  http://initech.zendesk.com/api/v2/users/43.json
               verified:  false

**** Result[16/26]
                    _id:  44
                 active:  false
                  alias:  Mr Gonzales
             created_at:  2016-06-08T10:26:06 -10:00
                  email:  gonzalesfloyd@flotonic.com
            external_id:  b5d38224-29c5-4aea-81a4-0da83d0a3f80
          last_login_at:  2014-11-04T08:34:49 -11:00
                 locale:  de-CH
                   name:  John Floyd
                  phone:  9894-382-253
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Dunlo
                tags[1]:  Greer
                tags[2]:  Crown
                tags[3]:  Strong
               timezone:  Hong Kong
                    url:  http://initech.zendesk.com/api/v2/users/44.json
               verified:  false

**** Result[17/26]
                    _id:  47
                 active:  false
                  alias:  Mr Doris
             created_at:  2016-05-17T04:06:23 -10:00
                  email:  dorisschultz@flotonic.com
            external_id:  3abbffbf-eb9d-4738-825b-cf3c3e76f215
          last_login_at:  2015-10-06T09:27:54 -11:00
                 locale:  en-AU
                   name:  Dawson Schultz
        organization_id:  122
                  phone:  9595-373-294
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  true
                tags[0]:  Linganore
                tags[1]:  Alleghenyville
                tags[2]:  Cataract
                tags[3]:  Woodruff
               timezone:  Martinique
                    url:  http://initech.zendesk.com/api/v2/users/47.json
               verified:  false

**** Result[18/26]
                    _id:  49
                 active:  true
                  alias:  Miss Jody
             created_at:  2016-05-12T08:39:30 -10:00
                  email:  jodyholcomb@flotonic.com
            external_id:  4bd5e757-c0cd-445b-b702-ee3ed794f6c4
          last_login_at:  2014-12-04T12:51:36 -11:00
                 locale:  zh-CN
                   name:  Faulkner Holcomb
        organization_id:  118
                  phone:  9255-943-719
                   role:  end-user
                 shared:  true
              signature:  Don't Worry Be Happy!
              suspended:  true
                tags[0]:  Hanover
                tags[1]:  Woodlake
                tags[2]:  Saticoy
                tags[3]:  Hinsdale
               timezone:  Antigua and Barbuda
                    url:  http://initech.zendesk.com/api/v2/users/49.json
               verified:  false

**** Result[19/26]
                    _id:  51
                 active:  true
                  alias:  Miss Esmeralda
             created_at:  2016-01-21T06:02:01 -11:00
                  email:  esmeraldabuckley@flotonic.com
            external_id:  4a4adf11-a0e8-461d-b310-0e52a052dc73
          last_login_at:  2015-03-01T03:06:05 -11:00
                 locale:  de-CH
                   name:  Green Buckley
        organization_id:  125
                  phone:  8154-223-420
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  true
                tags[0]:  Chumuckla
                tags[1]:  Dixie
                tags[2]:  Yardville
                tags[3]:  Riverton
               timezone:  Seychelles
                    url:  http://initech.zendesk.com/api/v2/users/51.json
               verified:  true

**** Result[20/26]
                    _id:  53
                 active:  false
                  alias:  Miss Middleton
             created_at:  2016-02-24T05:40:40 -11:00
                  email:  middletonlang@flotonic.com
            external_id:  99006a7e-3279-416d-a2f2-d92aca820ec0
          last_login_at:  2014-03-03T08:54:37 -11:00
                 locale:  en-AU
                   name:  Pena Lang
        organization_id:  113
                  phone:  9174-113-995
                   role:  end-user
                 shared:  false
              signature:  Don't Worry Be Happy!
              suspended:  false
                tags[0]:  Roderfield
                tags[1]:  Lemoyne
                tags[2]:  Grantville
                tags[3]:  Fredericktown
               timezone:  Liechtenstein
                    url:  http://initech.zendesk.com/api/v2/users/53.json
               verified:  true

   .done|.d to stop
   .quit|.q to exit
   or enter to continue

7 results remaining # 
   '.help|.h' to see example values
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a value to search Users[role] # 
   '.help|.h' to see available fields
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a term to search Users # 
   '.help|.h' to see example values
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a value to search Users[signature] # Searching 75 records for entries with 'signature' equal to 'Unicode ist spaß'
1 record(s) 

**** Result[1/1]
                    _id:  1
                 active:  true
                  alias:  Miss Coffey
             created_at:  2016-04-15T05:19:46 -10:00
                  email:  coffeyrasmussen@flotonic.com
            external_id:  74341f74-9c79-49d5-9611-87ef9b6eb75f
          last_login_at:  2013-08-04T01:03:27 -10:00
                 locale:  en-AU
                   name:  Francisca Rasmussen
        organization_id:  119
                  phone:  8335-422-718
                   role:  admin
                 shared:  false
              signature:  Unicode ist spaß
              suspended:  true
                tags[0]:  Springville
                tags[1]:  Sutton
                tags[2]:  Hartsville/Hartley
                tags[3]:  Diaperville
               timezone:  Sri Lanka
                    url:  http://initech.zendesk.com/api/v2/users/1.json
               verified:  true

End of results

   '.help|.h' to see example values
   '.back|.b' to go back
   '.quit|.q' to exit

Enter a value to search Users[signature] # 